ban/core/models.py

```diff
@@ -117,6 +117,7 @@ def housenumbers(self):
 
 
 class HouseNumber(Model):
+    CEA_FORMAT = '[\dAB]{2}\d{3}[234679ABCEGHILMNPRSTUVXYZ]{5}'
```

Comment:
Removal of the `5` but not the `S`? I'm surprised or I misinterpret the goal, a comment would be appreciated to explain the regexp a bit 😉